gamedata/alldefs_post.lua

```diff
@@ -239,27 +239,29 @@ function UnitDef_Post(name, uDef)
 
 		if modOptions.unbacom then	
 			if uDef.customparams.isunbacom or uDef.customparams.iscommander then
+
 				uDef.customparams.evolution_timer = modOptions.unbacomleveluprate*60
-			end
-			if  name == "armcom" then
+				if  name == "armcom" then
 				uDef.customparams.evolution_announcement = "Armada commanders have upgraded to level 2"
 				uDef.customparams.evolution_announcement_size = 18.5
 				uDef.customparams.evolution_target = "armcomlvl2"
 				uDef.customparams.evolution_condition = "timer"
-			end
-			if name == "corcom" then
+				uDef.movestate = 1
```

Comment:
Definitely do not change the default movestate for the normal com, it's very important that it be on hold position by default. I would rather this be solved by some kind of documentation by unit description or tooltip or something, having units change movestate without your input would be very annoying and could easily result in your com diving towards enemy defenses while you look away - maybe not a major issue for PvE where this will see most play, but still frustrating.

Another option would be having drones respond to an entirely new command. That would certainly be out of scope for this PR, and would likely be best handled as part of the more generic carrier behavior, but there is precedent for this with e.g. the hound weapon toggle. There's a gadget that handles that if you want to see how it's done.

Reply:
removed